assets/css/base/_sidebar.scss

```diff
@@ -44,7 +44,6 @@ $sidebar-width: 250px;
       }
 
       &:hover {
-        cursor: pointer;
```

Comment:
@annalickl @mfuerni merging this file was kinda like:

mhmm this look good and that looks good. Please check this file that I have not removed something that was actually wanted.